Automattic/wp-calypso
# ToS acceptance tracking: For the checkout page, remove legacy Business plan

#### Changes proposed in this Pull Request

* Since the Business plan is deprecated for new sites, this PR makes changes to add the Pro plan to cart, so that the ToS link is visible in the checkout page.

#### Testing instructions



* Follow the setup instructions in https://github.com/Automattic/wp-calypso/pull/62574 and verify that the checkout page screenshots are saved in the tos blog media directory. 



Related to #
